.github/workflows/docker-publish.yml

```diff
@@ -94,3 +94,74 @@ jobs:
         # This step uses the identity token to provision an ephemeral certificate
         # against the sigstore community Fulcio instance.
         run: echo "${TAGS}" | xargs -I {} cosign sign --yes {}@${DIGEST}
+
+
+  # build-glr:
```

Comment:
I've tried this as both 2 jobs and one, but they both had the same issue. I'll either re-enable as 2 jobs or delete these comments before final approval and merge